Makefile.in

```diff
@@ -62,6 +64,17 @@ LDFLAGS=-L. -Lopenbsd-compat/ @LDFLAGS@
 EXEEXT=@EXEEXT@
 MANFMT=@MANFMT@
 
+# *** ICL Modification ***
+SODIUM_VER = 1.0.10
```

Comment:
Just a quick note that they are on 1.0.11 now. Homebrew has that if you upgrade.